tendrl/node_agent/tests/test_definitions.py

```diff
@@ -0,0 +1,17 @@
+import pkgutil
+import yaml
+
+
+def test_valid_definitions(definitions_path):
```

Comment:
I would suggest rename the test something like `test_yaml_file_validity`

This and the proposed change of the fixture used here would make it run like this:

```
tendrl/node_agent/tests/test_definitions.py::test_yaml_file_validity[/../../../etc/tendrl_definitions_node_agent.yaml] PASSED
```

which would be much easier to immediatelly understand just by going through the test run report.